packages/mco/components/create-dr-policy/create-dr-policy.tsx

```diff
@@ -217,31 +260,82 @@ const CreateDRPolicy: React.FC<{}> = () => {
               {state.isClusterSelectionValid && (
                 <>
                   <FormGroup
-                    fieldId="selected-clusters"
-                    label={t('Selected clusters')}
+                    fieldId="select-backend"
+                    label={t('Select replication')}
                   >
-                    {state.selectedClusters.map((c, i) => (
-                      <SelectedClusterView
-                        key={getName(c)}
-                        index={i + 1}
-                        cluster={c}
-                      />
-                    ))}
+                    <FormHelperText>
+                      <HelperText className="mco-create-data-policy__text-input">
+                        <HelperTextItem>
+                          {t(
+                            'All disaster recovery prerequisites are met for both clusters. Multiple storage backends are available on both of the selected clusters.'
+                          )}
+                        </HelperTextItem>
+                      </HelperText>
+                    </FormHelperText>
+
+                    <SelectReplicationBackend
+                      selectedClusters={state.selectedClusters}
+                      dispatch={dispatch}
+                      selectedKey={state.replicationBackend}
+                    />
                   </FormGroup>
+                  {state.replicationBackend ===
+                    BackendType.NonDataFoundation && (
+                    <>
+                      <ThirdPartyStorageWarning docHref="www.redhat.com" />
```

Comment:
use `odfDRDocHome` as a placeholder until doc link is available or talk to doc team and get the exact url they will be using, they can add the content later, but have some agreement on the url...
```suggestion
                      <ThirdPartyStorageWarning docHref={odfDRDocHome(version)} />
```